src/plugins/phoenix.js

```diff
@@ -52,6 +52,10 @@ export default {
               return Promise.reject(new Error(response.statusText))
             })
             .catch(error => console.log(error))
+        },
+
+        isIE () {
+          return !!window.MSInputMethodContext && !!document.documentMode
```

Comment:
is this for any IE version of specific ones ? please add in a JSDoc block at least

Reply:
It checks only ie11. We don't support older version of ie since Phoenix wouldn't probably even get built there. I wrote the jsdod and renamed it also to `isIE11` to make it more prominent.

Reply:
thanks. it's just to be future proof, who knows what the next IE will be...